modules/Bio/EnsEMBL/Compara/PipeConfig/PairAligner_conf.pm

```diff
@@ -545,12 +545,19 @@ sub core_pipeline_analyses {
               -can_be_empty  => 1,
               -rc_name => $self->o('filter_duplicates_himem_rc_name'),
            },
- 	   {  -logic_name => 'update_max_alignment_length_after_net',
- 	      -module     => 'Bio::EnsEMBL::Compara::RunnableDB::GenomicAlignBlock::UpdateMaxAlignmentLength',
-	      -rc_name => '1Gb_job',
-	      -wait_for =>  [ 'create_filter_duplicates_net_jobs', 'filter_duplicates_net', 'filter_duplicates_net_himem' ],
+           {  -logic_name    => 'remove_partial_blocks',
+              -module        => 'Bio::EnsEMBL::Hive::RunnableDB::SqlCmd',
+              -parameters    => { 
+                                'sql' => "DELETE FROM genomic_align_block WHERE genomic_align_block_id NOT IN (SELECT genomic_align_block_id FROM genomic_align)"
```

Comment:
I'm wondering if doing `... (SELECT genomic_align_block_id FROM genomic_align GROUP BY genomic_align_block_id)` would make this query faster... (you could use `DISTINCT` instead, but I have read in several places that it is never faster than this alternative, surprisingly).

Reply:
I tried with `DISTINCT`, but was no faster and same for the `GROUP BY`, at least not noticeably :woman_shrugging: